pages/api_integration-full-node-streaming.md

```diff
@@ -31,19 +31,21 @@ To follow along with [Google's documentation on gRPC streaming clients](https://
 
 For Python, the corresponding code is already generated in [the v4-proto PyPi package](https://pypi.org/project/v4-proto/).
 
-## Maintaining Orderbook State 
+## Maintaining Orderbook and Subaccount State 
```

Comment:
update the first blurb on line 5 with subaccounts

Reply:
done